src/InventoryManager.cpp

```diff
@@ -39,17 +40,31 @@ int InventoryManager::userInput() {
             std::cin.clear();
             std::cin.ignore(10000, '\n');
 
-            std::cout << "Name Perishable/NonPerishable SubCategory Quantity ID SalePrice "
-                         "CostOfGood Tax ExpirationDate (Enter -1000000 if unsure): ";
-            std::cin >> name >> category >> sub_category >> quantity >> id >> sale_price >> buy_price >> tax >>
-                expiration;
+            std::cout << "Enter item name: ";
+            std::cin >> name;
+            std::cout << "Enter item category (Perishable or NonPerishable): ";
+            std::cin >> category;
+            std::cout << "Enter item sub-category: ";
+            std::cin >> sub_category;
+            std::cout << "Enter item quantity: ";
+            std::cin >> quantity;
+            std::cout << "Enter item id: (up to 6 digits) ";
+            std::cin >> id;
+            std::cout << "Enter sale price: (format xx.xx) $";
+            std::cin >> sale_price;
+            std::cout << "Enter purchase cost: (format xx.xx) $";
+            std::cin >> buy_price;
+            std::cout << "Enter item tax: (format xx.xx) $";
+            std::cin >> tax;
+            std::cout << "Enter expiration date: (format xx/xx/xxxx) ";
+            std::cin >> expiration;
 
             try {
-                if (category == "Perishable") {
-                    new_item = std::make_shared<PerishableItem>(name, category, sub_category, quantity, id, sale_price,
+                if (category == "Perishable" || "perishable") {
```

Comment:
You can't compare things like this -- this if statement is saying:

```if (category equals "Perishable" or the string literal perishable)```

And the string literal perishable part likely always  evaluates to true.

So, currently you can only add Perishable items.

Needs to be:
Here's what you could do instead:
```
/* From HelperFunctions.hpp
lowerCaseString(category);

if (category == "perishable") {
    ...
} else if (category == "nonperishable") {
    ...
}
```

Reply:
done!